hdr/sqlite_modern_cpp.h

```diff
@@ -413,12 +421,12 @@ namespace sqlite {
 		}
 
 		template <typename Function>
-		void define(const std::string &name, Function&& func) {
+		void define(const STR_REF &name, Function&& func) {
 			typedef utility::function_traits<Function> traits;
 
 			auto funcPtr = new auto(std::forward<Function>(func));
 			if(int result = sqlite3_create_function_v2(
-					_db.get(), name.c_str(), traits::arity, SQLITE_UTF8, funcPtr,
+					_db.get(), name.data(), traits::arity, SQLITE_UTF8, funcPtr,
```

Comment:
I do not think SQlite accepts not null-terminated function names because there is no way to communicate the length.

Reply:
I think we should stick to `.c_str()` here because it makes clearer that we really need a c-style string, especially the trailing nul byte.